src/tqec/interop/collada/read_write.py

```diff
@@ -62,15 +62,13 @@ def read_block_graph_from_dae_file(
     filepath: str | pathlib.Path,
     graph_name: str = "",
 ) -> BlockGraph:
-    """Read a Collada DAE file and construct a
-    :py:class:`~tqec.computation.block_graph.BlockGraph` from it.
+    """Read a Collada DAE file and construct a :py:class:`.BlockGraph` from it.
```

Comment:
I am confused, are we supposed to use `:class:` or `:py:class:`? In some of the earlier docstrings, the former was used. In this file, the latter is used.

Reply:
Both are fine, `:class:` is a shorthand for `:py:class:`. Let's try to use `:class:` everywhere for consistency.